code/modules/mob/living/simple_animal/hostile/retaliate/kangaroo.dm

```diff
@@ -21,6 +21,7 @@
 	atmos_requirements = list("min_oxy" = 5, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 2, "min_co2" = 0, "max_co2" = 5, "min_n2" = 0, "max_n2" = 0)
 	move_to_delay = 4 // at 20ticks/sec, this is 5 tile/sec movespeed, about the same as a 'fast human'.
 	speed = -1 // '-1' converts to 1.5 total move delay, or 6.6 tiles/sec movespeed
+	var/melee_damage_special = 60
```

Comment:
Undocumented change?

Reply:
No, yes, sort of. I added the variable so I could tweak it for subtypes / instances like Wally and be able to nerf the Kangaroo's (rather insane) base damage value for its special attack. 

The number was originally hardcoded and unmodifiable. I didn't change the value itself (Normal Kangaroos still wallop for 60 damage), only added the variable.